Fable.Remoting.MsgPack/Write.fs

```diff
@@ -228,6 +230,16 @@ let inline writeGuid (g: Guid) out =
 let inline writeBigInteger (i: bigint) out =
     writeBin (i.ToByteArray ()) out
 
+let inline writeDataTable (dt: System.Data.DataTable) out =
+  if String.IsNullOrWhiteSpace dt.TableName then
+    failwithf "datatable must have TableName property set"
+  do
```

Comment:
Why `do`? :)

Reply:
> if String.IsNullOrWhiteSpace dt.TableName then

Doesn't this sound like an application specific logic and not so much a data transport concern or am I missing something? 🤔

Reply:
@kerams it is useful to scope the `use` of stuff and signal 'side-effectey' block, but the code was pasted there and I didn't notice in the short body, it doesn't matter.

@Zaid-Ajaj I've removed it, I was actually dealing with having to debug the server side and it was just the intent to help user somehow, but it can be done outside the library, no problem. I've updated the code.